{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["poetry>=1.0"]
-build-backend = "poetry.masonry.api"
+requires = ["poetry-core>=1.1.0"]
```

Comment:
We would also need to remove our `importlib-metadata` hacks from packaging files.

Reply:
I've changed version and removed `importib-metadata` version pinning.